hibernate-core/src/main/java/org/hibernate/query/sqm/sql/internal/AssignableSqmPathInterpretation.java

```diff
@@ -1,31 +0,0 @@
-/*
```

Comment:
I guess this is the interface you mentioned to me on Zulip?  I'm surprised it is not used, but then again I forget exactly how I implemented this part of UPDATE HQL/Criteria handling.

Reply:
yes this is the one I mentioned you